osquery/database/plugins/rocksdb.cpp

```diff
@@ -254,6 +254,16 @@ Status RocksDBDatabasePlugin::get(const std::string& domain,
   return Status(s.code(), s.ToString());
 }
 
+Status RocksDBDatabasePlugin::get(const std::string& domain,
+                                  const std::string& key,
+                                  int& value) const {
+  std::string result;
+  auto s = this->get(domain, key, result);
+  if (s.ok()) {
+    value = std::stoi(result);
```

Comment:
Could it be a wrong string representation of int? In this case `std::stoi` throws an exception which we should handle here.

Reply:
Good catch, it should not be wrong, but of course, need to add checks.

Reply:
Actually, will leave it like that, this method should be changed.